src/components/NimiqCheckoutOption.vue

```diff
@@ -380,8 +404,17 @@ export default class NimiqCheckoutOption
     }
 
     private get hasSufficientBalanceAccount(): boolean {
```

Comment:
The function name is now not accurate anymore, since it also tests for eligibility with the recipient address. So can be renamed to `hasEligibleAddress()`.